drivers/disk/Kconfig.sdmmc

```diff
@@ -45,7 +45,7 @@ config SDMMC_STM32
 	select USE_STM32_HAL_MMC_EX if SDMMC_STM32_EMMC && SOC_SERIES_STM32L4X
 	select USE_STM32_LL_SDMMC
 	select USE_STM32_HAL_DMA if (SOC_SERIES_STM32L4X || SOC_SERIES_STM32F7X || SOC_SERIES_STM32F4X)
-	select DMA if ($(DT_STM32_SDMMC_HAS_DMA) && SOC_SERIES_STM32F4X)
+	select DMA if $(DT_STM32_SDMMC_HAS_DMA) && (SOC_SERIES_STM32L4X || SOC_SERIES_STM32F7X)
```

Comment:
Not sure why the F4 disappeared here.

Reply:
That was a mistake, F4 got replaced with L4. Fixed it.